pages/pixel-ui/avatar.mdx

```diff
@@ -0,0 +1,22 @@
+import MethodList from "/components/method_list";
+
+# Avatar
+## Methods
+The following vgui methods are accessible via the `PANEL` table:
+<MethodList methods={[
+	{
+		name: "SetSteamID",
+		parent: "Avatar:",
+		description: "Loads avatar by its 64-bit Steam ID (community ID).",
+		args: ["string", "number"],
+		href: "avatar/set-steamid"
+	},
+	{
+		name: "SetPlayer",
+		parent: "Avatar:",
+		description: "Loads avatar for a given player.",
+		args: ["Player", "number"],
+		href: "avatar/set-player"
+	},
+

```

Comment:
Get/SetMaskSize should also be documented, see the following source:
https://github.com/TomDotBat/pixel-ui/blob/master/lua/pixelui/elements/cl_avatar.lua#L20